readme.md

```diff
@@ -78,11 +106,22 @@ var Panda = function(name, age) {
   this.age = age;
   this.num_bamboo_eaten = 0;
 }
+
+Panda.prototype = {
+  eat_bamboo: function(){
+    self = this;
+    var eaten = self.num_bamboo_eaten;
+    self.num_bamboo_eaten = eaten + 1;
+  }
+}
+
 ```
 
 Your Answer:
 ```text
 
+So here, I added a prototype after the constructor method that adds one to num_bamboo_eaten
+
 ```
 
 
```

Comment:
Let me know if you have any questions about the remaining quiz questions you didn't get to!